### PR TITLE
Update hybrid-application.md

### PR DESCRIPTION
**Pull request by DongHyunSong**
Add note for TV profile.

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Comment by DongHyunSong** [comment]
Hi. I am WebRuntime developer in VD division, we want to add a "note" that TV profile doesn't support Hybrid application type.

**Comment by mijin85cho** [comment]
> Hi. I am WebRuntime developer in VD division, we want to add a "note" that TV profile doesn't support Hybrid application type.

Thanks for the update, @DongHyunSong 
@asm-foysal Would you please assign this PR to a reviewer?

File: docs/application/dotnet/get-started/hybrid-application.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ You can create a hybrid application package by merging Web application and .NET
 
 > [!NOTE]
 > Currently, only command-line interface is provided for .NET and Web hybrid application package.
```
Comment (omar-srbd) [inline docs/application/dotnet/get-started/hybrid-application.md:10]:
```suggestion
> Currently, only command-line interface is provided for .NET and web hybrid application package.
```

File: docs/application/dotnet/get-started/hybrid-application.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ You can create a hybrid application package by merging Web application and .NET
...
+> Hybrid application is not supported in TV profile officially.
```
Comment (omar-srbd) [inline docs/application/dotnet/get-started/hybrid-application.md:11]:
```suggestion
> Hybrid application is not supported in the TV profile officially.
```

**Review by omar-srbd — COMMENTED** [review]
Hello @DongHyunSong,
I hope you are doing well. I have reviewed the file on this PR and made some in-line suggestions where I could. However, there were some instances where I could not add suggestion.
I am listing them below:
1.	_Line 3:_ “…application and Web application.”
_Suggestion:_ “…application and web application.”
2.	_Line 5:_ “…develop the .NET application using Visual Studio and Web application…This implies that, you…”
_Suggestion:_ “…develop .NET applications using Visual Studio and web applications…This implies that you…”
3.	_Line 7:_ “…merging Web application…”
_Suggestion:_ “…merging web application…”
4.	_Line 17:_ “The Web application package (wgt) file…Web Widget application and the Web UI application…Web hybrid…”
_Suggestion:_ “The web application package (wgt) file… web widget application and the web UI application…web hybrid…”
5.	_Line 19:_ “The .NET and Web hybrid application…”
_Suggestion:_ “The .NET and web hybrid application…”
6.	_Line 21:_ “The .NET and Web hybrid application…type of a merge between…Web application…Package ID of the Web application generates the Package ID…”
_Suggestion:_ “The .NET and web hybrid application…type of merge between…web application…package ID of the web application generates the package ID…”
7.	_Line 34:_ “…Web Widget sample application.”
_Suggestion:_ “…web widget sample application.”
8.	_Line 44:_ “…the Web application:”
_Suggestion:_ “…the web application:”
9.	**NOTE:** Lists must be introduced with a  complete sentence. For the listing that follows the heading "Example" no such complete sentence has been used. Please add an introductory sentence before the list.
**Reference:** https://developers.google.com/style/lists#introductory-sentences-for-lists

**Comment by mijin85cho** [comment]
Hello, @omar-srbd Hope you're doing well.

@DongHyunSong is in VD division, that means this page is not under his responsibility.
By him, only a note related with the VD division is added in this PR.
So I suggest checking only the updated part.

**Comment by omar-srbd** [comment]
Hello @mijin85cho, I am doing well, Thank you. 
I hope the same for you.
Thank you for notifying the matter to me. :)


@DongHyunSong, please ignore the suggestions that have been made on lines other than the line that you have added (_i.e._ **Line 11**) and implement the change at your earliest convenience.
Thank you.

**Comment by DongHyunSong** [comment]
> Hello @mijin85cho, I am doing well, Thank you. I hope the same for you. Thank you for notifying the matter to me. :)
> 
> @DongHyunSong, please ignore the suggestions (_[#1648 (review)](https://github.com/Samsung/tizen-docs/pull/1648#pullrequestreview-1009976914)_ & [#1648 (comment)](https://github.com/Samsung/tizen-docs/pull/1648#discussion_r899746395)) that have been made on lines other than the line that you have added (_i.e._ **Line 11**, [#1648 (comment)](https://github.com/Samsung/tizen-docs/pull/1648#discussion_r899746884)) and implement the small change on the modified part at your earliest convenience. Thank you.

It is fine that even though my commit is not applied, but please add a note that TV profile doesn't support Hybrid application.

**Comment by mijin85cho** [comment]
@asm-foysal When this PR is merged, please raise a PR to sync live branch to master.

**Review by asm-foysal — COMMENTED** [review]
@mijin85cho Sure, we'll create a sync master to live PR once this PR has been merged.

**Review by omar-srbd — COMMENTED** [review]
@DongHyunSong, thanks for implementing the change as per suggestion.

**Review by omar-srbd — APPROVED** [review]
@mijin85cho @asm-foysal, I am approving this PR for merging.

**Pull request merged**